UPGRADE.md

```diff
@@ -2,6 +2,10 @@
 
 ## 2.1.9
 
+### A new argument `$requestStack` has been added to the `ContentTwigExtension`
+
+Instantiating a `ContentTwigExtension` without the `$requestStack` argument is deprecated.
+
```

Comment:
Why did you add this? It's already [here](https://github.com/sulu/sulu/blob/2.1/UPGRADE.md)

Reply:
Ah, this is not the only wrong thing in here 😂